synapse/python_dependencies.py

```diff
@@ -43,7 +43,6 @@
         "matrix_angular_sdk>=0.6.8": ["syweb>=0.6.8"],
     },
     "preview_url": {
-        "lxml>=3.6.0": ["lxml"],
```

Comment:
Do we say what version of lxml is needed anywhere?
@ara4n: What's the rational for requiring ``>=3.6.0`` ooi?